src/SearchHeader.tsx

```diff
@@ -9,62 +9,69 @@ import { EMPTY } from './constants';
 import RegistryButton from './RegistryButton';
 
 export interface SearchHeaderProps {
-    search(searchValue: string): object[];
-    registryName: string;
+	search(searchValue: string): object[];
+	registryName: string;
 }
 
 const SearchHeader = (props: SearchHeaderProps): any => {
-    
-    const { search, registryName } = props;
-    
-    const [searchValue, setSearchValue] = useState(EMPTY);
-    
-    const handleSearchValueChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
-        setSearchValue(event.target.value);
-    }
-    
-    const searchThis = () => {
-        search(searchValue);
-    };
-    
-    const SearchButton = withRouter(
-        ({history}: any) =>
-            <Button                    
-                onClick={() => {
-                    searchThis();
-                    history.push("/".concat(registryName, "/Search"));
-                }}
-                color="inherit"
-            >Search</Button>
-    );
+	const { search, registryName } = props;
 
-    const EditButton = withRouter(
-        ({history}: any) =>
-            <Button onClick={() => history.push("/".concat(registryName))}>Edit</Button>
-    );
-    
-    return (<div>
-        <AppBar position="static" color="default">
-            <Toolbar variant="dense">
-                <Typography variant="h6" color="inherit" align="left">
-                Search ({registryName})
-                </Typography>
-                <EditButton />
-                <RegistryButton />
-                <TextField
-                    variant='outlined'
-                    margin='normal'
-                    name='search'
-                    label='Search'
-                    type='search'
-                    id='search'
-                    autoFocus
-                    onChange={handleSearchValueChange}
-                />
-                <SearchButton/>
-            </Toolbar>
-        </AppBar>
-    </div>);
-}
+	const [searchValue, setSearchValue] = useState(EMPTY);
+
+	const handleSearchValueChange = (event: React.ChangeEvent<HTMLInputElement>): void => {
+		setSearchValue(event.target.value);
+	};
+
+	const searchThis = () => {
+		search(searchValue);
+	};
+
+	const SearchButton = withRouter(({ history }: any) => (
+		<Button
+			onClick={() => {
+				searchThis();
+				history.push('/'.concat(registryName, '/Search'));
```

Comment:
We do history.push('/'.concat(registryName), {something}) a lot, can it be generalised?